server/handlebars/embed/views/main.hbs

```diff
@@ -9,7 +9,7 @@
     {{#unless dataEmpty}}
         {{#if (isCandidate phase)}}
         <details id="embed-report-phase-container">
-            <summary id="candidate-title"><span>Warning! Unapproved Report</span></summary>
+            <summary id="candidate-title"><span><span>Warning! Unapproved Report</span></span></summary>
```

Comment:
I think this could be improved semantically with a heading and a span instead of two spans.

Reply:
This has been taken care of.